cmd/podman/images/scan.go

```diff
@@ -0,0 +1,158 @@
+package images
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/containers/common/pkg/completion"
+	"os"
+	"text/template"
+
+	"github.com/containers/podman/v2/cmd/podman/common"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/libpod/define"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/containers/podman/v2/pkg/specgen"
+	"github.com/spf13/cobra"
+)
+
+var (
+	imageScanCommand = &cobra.Command{
+		Use:     "scan [run-options] IMAGE [scanner-options]",
+		Short:   "Scan a container image for known vulnerabilities",
+		Long:    `Scan a container image for known vulnerabilities. The image name or digest can be used.`,
+		Args:    cobra.MinimumNArgs(1),
+		RunE:    scan,
+		Example: `podman image scan centos:latest`,
+	}
+
+	scanOptions entities.ImageScanOptions
+)
+
+func scanFlags(cmd *cobra.Command) {
+	flags := cmd.Flags()
+
+	flagName := "scanner"
+	flags.StringVarP(
+		&scanOptions.ScannerImage,
+		flagName, "s", "docker.io/anchore/syft:latest",
```

Comment:
Can we pin it to a specific version of `syft` in contrast to using `latest`?  `latest` is a moving target which makes it hard to test, verify and rely on.